src/app/mod.rs

```diff
@@ -1227,6 +1230,32 @@ impl<'a, 'b> App<'a, 'b> {
         self.get_matches_from_safe(&mut env::args_os())
     }
 
+    /// Similar to [`App::get_matches`] but also reads args from
+    /// env-var `env_var_name`.
+    ///
+    /// Defaults to read from "VAR" if `env_var_name` is `None`.
+    pub fn get_matches_with_env(self, env_var_name: Option<String>)
```

Comment:
Ahh right, I've used the Into-trait once or twice before. I always have to lookup the syntax though :)